app/views/content/is-teaching-right-for-me/maths/_article.html.erb

```diff
@@ -5,7 +5,7 @@
     </p>
 
     <p>
-      An exciting career spent passing your knowledge on to the next generation could be for you.
+      As a secondary maths teacher, an exciting career spent passing your knowledge on to the next generation could be for you.
     </p>
 <p>Tax-free bursaries of £28,000 or scholarships of £30,000 are available for eligible trainee maths teachers.</p>
   </section>
```

Comment:
Line 23 - I don't think this makes a lot of sense (I raised this on the original PR):

<p>A good understanding of maths has a lot of benefits for young people to prepare them for their future, **like using financial maths and data handling**.</p>

Can we talk about `managing their personal finances and handling data` (can probably be improved!)